pipelines/sqre/infra/build_sciplatlab.groovy

```diff
@@ -58,13 +58,42 @@ notify.wrap {
         conn.doOutput = true
         conn.outputStream << json.toPrettyString()
 
-        println("responseCode: ${conn.responseCode}")
-        def text = conn.getInputStream().getText()
-        println("response: ${text}")
+        assert (${conn.responseCode} == 204) : "API dispatch failed: ${conn.responseCode}|${conn.getInputStream().getText()}"
       }
-    }
-  } // run
+    } // stage
+
+    stage('poll for job completion at GHA') {
+      // We presume that GH Actions will continue to list most recent runs
+      // at the top.  Poll the build.yaml runs for the first result.
+
+      // I think we always want the prod branch.
 
+      def url = new URL("https://api.github.com/repos/lsst-sqre/sciplat-lab/actions/workflows/build.yaml/runs?per_page=1%25branch=prod")
```

Comment:
Parameter separator is `&`, not URL-encoded, I believe.  `%25` (`%`) is definitely wrong.